sql/table.go

```diff
@@ -97,6 +97,24 @@ func makeTableDesc(p *parser.CreateTable) (structured.TableDescriptor, error) {
 	return desc, nil
 }
 
+func (p *planner) getTableDesc(qname parser.QualifiedName) (
+	*structured.TableDescriptor, error) {
+	var err error
+	qname, err = p.normalizeTableName(qname)
+	if err != nil {
+		return nil, err
+	}
+	dbDesc, err := p.getDatabaseDesc(qname.Database())
+	if err != nil {
+		return nil, err
+	}
+
+	nameKey := keys.MakeNameMetadataKey(dbDesc.ID, qname.Table())
+	desc := structured.TableDescriptor{}
+	err = p.getDescriptor(nameKey, &desc)
```

Comment:
can this be `return &desc, p.getDescriptor(nameKey, &desc)`? the assignment here will result in a legitimate shadowing warning.